tests/indent/common.lua

```diff
@@ -84,6 +96,7 @@ function M.run_indent_test(file, runner, opts)
 
   -- load reference file
   vim.cmd(string.format("edit %s", file))
+  vim.bo.indentexpr = "nvim_treesitter#indent()"
```

Comment:
this should not be necessary! But otherwise Rust and Lua indentexpr from ft-plugin would load!